### PR TITLE
=str ActorPublisher mustn't to signal onSubscribe on the given subscriber more than once.

### PR DESCRIPTION
**Pull request by MQ-EL**
Hi, does it need to open an issue?

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
@MQ-EL yes, please open a ticket describing the issue, thanks!

**Comment by drewhk** [comment]
LGTM

**Comment by MQ-EL** [comment]
refs #20740

**Comment by johanandren** [comment]
Thanks, LGTM!

**Comment by MQ-EL** [comment]
:)

File: akka-stream/src/main/scala/akka/stream/actor/ActorPublisher.scala [hunk 1]
```diff
@@ -302,11 +302,10 @@ trait ActorPublisher[T] extends Actor {
           tryOnSubscribe(sub, CancelledSubscription)
           tryOnComplete(sub)
         case Active | Canceled ⇒
-          tryOnSubscribe(sub, CancelledSubscription)
-          tryOnError(
-            sub,
-            if (subscriber == sub) ReactiveStreamsCompliance.canNotSubscribeTheSameSubscriberMultipleTimesException
-            else ReactiveStreamsCompliance.canNotSubscribeTheSameSubscriberMultipleTimesException)
+          if (subscriber == sub)
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/actor/ActorPublisher.scala]: Per spec this should be eq I believe?

Comment (drewhk) [inline akka-stream/src/main/scala/akka/stream/actor/ActorPublisher.scala]: yes, right

**Comment by ktoso** [comment]
LGTM after https://github.com/akka/akka/pull/20733/files/219caf919f5e9c80a9a1535b63c52cb9dd0f2ec1#r66052924 changed :)

Thanks, very nice catch!

File: akka-stream/src/main/scala/akka/stream/actor/ActorPublisher.scala (hunk 1 above)
Comment (MQ-EL) [inline akka-stream/src/main/scala/akka/stream/actor/ActorPublisher.scala]: yes，i think you are right. will change it

**Comment by ktoso** [comment]
LGTM, thanks!

**Pull request merged**